node/pkg/watchers/evm/chain_config.go

```diff
@@ -282,3 +282,28 @@ func (w *Watcher) verifyEvmChainID(ctx context.Context, logger *zap.Logger, url
 
 	return nil
 }
+
+// GetContractAddr returns the configured contract address for the specified environment / chain.
```

Comment:
The name is wrong.

Reply:
Will fix the comment in the next round of changes.